tests/models/test_hf_model.py

```diff
@@ -457,3 +457,49 @@ def test_hf_loading_errors(tiny_bert_model, tiny_bert_tokenizer, model_class_nam
     with error_contexts[model_class_name]:
         _, _ = HuggingFaceModel.hf_from_composer_checkpoint(str(tmp_path / 'hf-checkpoint.pt'),
                                                             model_class_name_to_class[model_class_name])
+
+
+@pytest.mark.parametrize('model_type', ['gpt', 'bert'])
```

Comment:
Ah, sorry i told you to use the wrong thing for fixtures. unfortunately pytest does not allow you to put fixtures in the parametrization as you may have discovered, so instead you can use `configure_tiny_bert_model` and `configure_tiny_gpt_model` (again pattern match to create the gpt one), found in https://github.com/mosaicml/composer/blob/08f43a489b5d6cd04506c91bb8cbc52b06ebb296/tests/common/models.py#L376. example of doing this here: https://github.com/mosaicml/composer/blob/7d400151dcf14b497741ec427bfbc2345439b598/tests/algorithms/test_gated_linear_units.py#L70-L78